src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -129,6 +133,19 @@ export default class rd2EntryForm extends LightningElement {
 
     @track error = {};
 
+    @api 
+    get nextDonationDate() {
+        const localDate = new Date(this._nextDonationDate);
+        return new Date(localDate.getUTCFullYear(), localDate.getUTCMonth(), localDate.getUTCDate());
+    }
+
+    @api 
+    get cardExpDate(){
```

Comment:
Does normal edit/insert still work if the User does not have edit permissions to these fields?

Reply:
Yes, but I think we may want to rethink how permissions are being checked in regard to this widget.